telegram/auth/qrlogin/qrlogin.go

```diff
@@ -163,6 +185,13 @@ func (q QR) Auth(
 			if err != nil {
 				return nil, err
 			}
+
+			// If empty token, it means AuthLoginTokenSuccess was returned
+			if t.String() == "" {
+				// QR was scanned and accepted, break out of loop
+				break
+			}
```

Comment:
This `break` would break out of `select`, not out of the `for` loop.

The logic itself seems fine to me, but comment should be fixed.